cmd/webhook/main.go

```diff
@@ -220,6 +222,7 @@ func main() {
 		Port:        8443,
 		SecretName:  "webhook-certs",
 	})
+	ctx = injection.WithNamespaceScope(ctx, system.Namespace())
```

Comment:
This is a bandaid, which won't work for bindings (e.g. in eventing).

We will need to code-gen resource specific versions and use it for secrets and configmaps in those contexts.

This helper was intended to enable running *entire controllers* scoped to a namespace.

Reply:
Correct. But this will only solve serving WH.